conformance/go.mod

```diff
@@ -3,21 +3,23 @@ module github.com/opencontainers/distribution-spec/conformance
 go 1.17
 
 require (
-	github.com/bloodorangeio/reggie v0.5.0
+	github.com/bloodorangeio/reggie v0.6.0
 	github.com/google/uuid v1.3.0
 	github.com/onsi/ginkgo v1.16.5
-	github.com/onsi/gomega v1.16.0
+	github.com/onsi/gomega v1.27.4
```

Comment:
We need to update the Dockerfile for this change. We currently use Go 1.17, and this change requires at least 1.18. The following worked for my tests:

```
$ git diff
diff --git a/conformance/Dockerfile b/conformance/Dockerfile
index c462516..249c203 100644
--- a/conformance/Dockerfile
+++ b/conformance/Dockerfile
@@ -1,7 +1,7 @@
 # ---
 # Stage 1: Install certs and build conformance binary
 # ---
-FROM docker.io/golang:1.17.1-alpine3.14 AS builder
+FROM docker.io/golang:1.19.7-alpine3.17 AS builder
 ARG VERSION=unknown
 ARG GO_PKG=github.com/opencontainers/distribution-spec/conformance
 RUN apk --update add git make ca-certificates && mkdir -p /go/src/${GO_PKG}
```

Reply:
missing dockerfile change?

Reply:
@imjasonh I'm not seeing the change, give it another git push?